spec/acceptance/suites/tar_el6/files/simp_conf.yaml.erb

```diff
@@ -0,0 +1,270 @@
+---
+#========================================================================
+# simp config answers
+#
+# Generated for simp scenario on 2017-04-04 16:00:21
+# using simp-cli version 3.0.0
+#------------------------------------------------------------------------
+# You can use these answers to quickly configure subsequent
+# simp installations by running the command:
+#
+#   simp config -A /PATH/TO/THIS/FILE
+#
+# simp config will prompt for any missing items.
+#
+# NOTE: All YAML keys that begin with 'cli::' are used
+# by simp config, internally, and are not Puppet hieradata.
+#========================================================================
+
+# === cli::is_ldap_server ===
+# Whether the SIMP server will also be the LDAP server.
+#
+cli::is_ldap_server: true
+
+# === cli::network::dhcp ===
+# Whether to use DHCP to set up your network ("static" or "dhcp").
+cli::network::dhcp: dhcp
+
+# === cli::network::gateway ===
+# The default gateway.
+cli::network::gateway: 10.0.2.2
```

Comment:
These addresses are not guaranteed unless you add them into your nodeset and, even then, bad things might happen.

These should be discovered and populated from the environment.